custom-words.txt

```diff
@@ -511,6 +511,7 @@ exportconfiguration
 exportdevices
 exporterrors
 exportstatus
+Exprired
```

Comment:
> ?

backend team said it's typo but they can't change it because it will cause breaking change. So in order to pass validation i have to add this as custom word

Reply:
This review hasn't been signed off on yet, so I don't see how it's possible to have a breaking change. Please ask the backend team to fix this.

Reply:
@fengzhou-msft, @yungezz is there any way to merge this PR in without adding `Exprired` to the custom-words list?

Reply:
> @fengzhou-msft, @yungezz is there any way to merge this PR in without adding `Exprired` to the custom-words list?

Hi @anthony-c-martin  I have removed `Exprired` from the custom-words list and added it to cSpell.json to suppress spell check